src/sentry/mediators/sentry_apps/creator.py

```diff
@@ -58,6 +60,8 @@ def _create_sentry_app(self):
             overview=self.overview,
         )
 
+        return self.sentry_app
```

Comment:
I don't think you need this because we already have `self.app`, maybe its worth changing that to be `self.sentry_app`?